web/profiles/custom/yalesites_profile/modules/custom/ys_views_basic/ys_views_basic.module

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * @file
+ * Contains ys_views_basic.module functions.
+ */
+
+/**
+ * Add template file for views basic parameters.
+ *
+ * Only used for preview on back-end.
+ *
+ * Implements hook_theme().
+ */
```

Comment:
Super minor formatting thing. I think hooks are only supposed to have ' Implments...' line. Any code comments can be placed as inline comments within the function

Reply:
Moved into inline comment.